API.yaml

```diff
@@ -489,6 +489,92 @@ paths:
             application/json:
               schema:
                 type: string
+ /experiments/{expUUID}/annotations:
```

Comment:
We should keep this endpoint, but it should be use to create annotations (bookmark). Right now it is used to fetch annotations, I think we can say that this endpoint is only to manage "global" annotations and each output have the responsibility to return these annotations when you query the specific annotations for this output.